addons/mail/static/src/core/web/messaging_menu.xml

```diff
@@ -108,7 +108,7 @@
         </div>
     </div>
     <div t-if="ui.isSmall" class="o-mail-MessagingMenu-navbar d-flex border-top bg-view shadow-lg w-100 btn-group">
-        <button t-foreach="tabs" t-key="tab.id" t-as="tab" class="o-mail-MessagingMenu-tab btn d-flex flex-column align-items-center flex-grow-1 flex-basis-0 p-2 mx-0" t-att-class="{
+        <button t-foreach="tabs" t-key="tab.id" t-as="tab" class="o-mail-MessagingMenu-tab btn d-flex flex-column align-items-center flex-grow-1 flex-basis-0 p-2 mx-0 border-0" t-att-class="{
```

Comment:
Might be working to use `.btn-link` instead of removing the border of `.btn` with `border-0`. It's inefficient to add and then remove stuff.

Reply:
`btn-link` and `text-primary` does not result to the expected styling:
![Screenshot 2024-06-24 at 15 44 47](https://github.com/odoo/odoo/assets/6569390/bcd2e9f3-8ac8-4ad7-81c8-3625bace80ee)
We want the primary tint so purple. If we go with `btn-link` then we need to add an `!important` in CSS, which results to more complex code than just canceling the `border-0`.

Reply:
You can achieve this without a button them:

Remove the button, use `<a role="button">`, and remove the `btn` class border dans stuff. You keep the `text-primary`: 

![image](https://github.com/odoo/odoo/assets/1810149/b29abb4b-9bab-4f62-98ca-26a1e293f043)
![image](https://github.com/odoo/odoo/assets/1810149/f631a55a-4a27-49f5-ab0d-8674a6d5ac6e)

Reply:
You can use a text-reset class to eliminate the green color of chat and channel, but IMHO, the green is better in dark mode.